app/models/concerns/foreman_git_templates/host_extensions.rb

```diff
@@ -18,6 +18,9 @@ def git_template_tmpfile
       return unless git_template_url
 
       @git_template_tmpfile ||= RepositoryFetcher.call(git_template_url)
+    rescue ::ForemanGitTemplates::RepositoryFetcher::RepositoryFetcherError => e
+      Foreman::Logging.exception(“GitTemplates: Failed to fetch data from #{git_template_url}, e)
```

Comment:
```suggestion
      Foreman::Logging.exception("GitTemplates: Failed to fetch data from #{git_template_url}", e)
```

^^ happens when you try to code on an iPad ;-)